### PR TITLE
Irm-10 NUTNR predeploys

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Glider 575 and SUMO-10 NUTNR predeploys.

Note: There are 2 csv files for the glider NUTNR S/N 727.  No vendor cal on file that matches the earlier .cal file in the test results (and no existing cal csvs for this SN) so I created cal csvs for both the old & new .cal files that were part of the predeploy testing.

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01086__20230830.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
....56, 273.37, 274.17, 274.98, 275.78, 276.59, 277.39, 278.2, 279.01, 279.81, 280.62, 281.42, 282.23, 283.03, 283.84, 284.64, 285.45, 286.25, 287.06, 287.86, 288.67, 289.47, 290.28, 291.08, 291.89, 292.69, 293.5, 294.3, 295.11, 295.91, 296.72, 297.52, 298.33, 299.13, 299.94, 300.74, 301.55, 302.35, 303.15, 303.96, 304.76, 305.57, 306.37, 307.17, 307.98, 308.78, 309.58, 310.39, 311.19, 311.99, 312.8, 313.6, 314.4, 315.21, 316.01, 316.81, 317.61, 318.42, 319.22, 320.02, 320.82, 321.63, 322.43, 323.23, 324.03, 324.83, 325.63, 326.44, 327.24, 328.04, 328.84, 329.64, 330.44, 331.24, 332.04, 332.84, 333.64, 334.44, 335.24, 336.04, 336.84, 337.64, 338.44, 339.24, 340.04, 340.84, 341.63, 342.43, 343.23, 344.03, 344.83, 345.63, 346.42, 347.22, 348.02, 348.81, 349.61, 350.41, 351.21, 352, 352.8, 353.59, 354.39, 355.19, 355.98, 356.78, 357.57, 358.37, 359.16, 359.96, 360.75, 361.54, 362.34, 363.13, 363.92, 364.72, 365.51, 366.3, 367.1, 367.89, 368.68, 369.47, 370.26, 371.06, 371.85, 372.64, 373.43, 374.22, 375.01, 375.8, 376.59, 377.38, 378.17, 378.96, 379.75, 380.54, 381.32, 382.11, 382.9, 383.69, 384.47, 385.26, 386.05, 386.84, 387.62, 388.41, 389.19, 389.98, 390.76, 391.55, 392.33, 393.12]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01086__20230830.csv:8]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01087__20230830.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
..., 273.4, 274.2, 275, 275.8, 276.61, 277.41, 278.21, 279.01, 279.81, 280.62, 281.42, 282.22, 283.02, 283.83, 284.63, 285.43, 286.23, 287.03, 287.84, 288.64, 289.44, 290.24, 291.04, 291.85, 292.65, 293.45, 294.25, 295.05, 295.85, 296.66, 297.46, 298.26, 299.06, 299.86, 300.66, 301.46, 302.27, 303.07, 303.87, 304.67, 305.47, 306.27, 307.07, 307.87, 308.67, 309.47, 310.28, 311.08, 311.88, 312.68, 313.48, 314.28, 315.08, 315.88, 316.68, 317.48, 318.28, 319.08, 319.88, 320.68, 321.48, 322.27, 323.07, 323.87, 324.67, 325.47, 326.27, 327.07, 327.87, 328.67, 329.46, 330.26, 331.06, 331.86, 332.66, 333.45, 334.25, 335.05, 335.85, 336.64, 337.44, 338.24, 339.03, 339.83, 340.63, 341.42, 342.22, 343.02, 343.81, 344.61, 345.4, 346.2, 346.99, 347.79, 348.58, 349.38, 350.17, 350.97, 351.76, 352.56, 353.35, 354.14, 354.94, 355.73, 356.53, 357.32, 358.11, 358.9, 359.7, 360.49, 361.28, 362.07, 362.86, 363.66, 364.45, 365.24, 366.03, 366.82, 367.61, 368.4, 369.19, 369.98, 370.77, 371.56, 372.35, 373.14, 373.93, 374.72, 375.51, 376.29, 377.08, 377.87, 378.66, 379.45, 380.23, 381.02, 381.81, 382.59, 383.38, 384.16, 384.95, 385.74, 386.52, 387.31, 388.09, 388.88, 389.66, 390.44, 391.23, 392.01, 392.79]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01087__20230830.csv:8]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00727__20200103.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...3.18, 273.99, 274.79, 275.6, 276.41, 277.21, 278.02, 278.82, 279.63, 280.44, 281.24, 282.05, 282.86, 283.67, 284.47, 285.28, 286.09, 286.89, 287.7, 288.51, 289.31, 290.12, 290.93, 291.73, 292.54, 293.35, 294.16, 294.96, 295.77, 296.58, 297.38, 298.19, 299, 299.8, 300.61, 301.42, 302.23, 303.03, 303.84, 304.65, 305.45, 306.26, 307.07, 307.87, 308.68, 309.49, 310.29, 311.1, 311.91, 312.71, 313.52, 314.33, 315.13, 315.94, 316.75, 317.55, 318.36, 319.17, 319.97, 320.78, 321.58, 322.39, 323.2, 324, 324.81, 325.61, 326.42, 327.23, 328.03, 328.84, 329.64, 330.45, 331.25, 332.06, 332.86, 333.67, 334.47, 335.28, 336.08, 336.89, 337.69, 338.5, 339.3, 340.11, 340.91, 341.71, 342.52, 343.32, 344.13, 344.93, 345.73, 346.54, 347.34, 348.14, 348.95, 349.75, 350.55, 351.35, 352.16, 352.96, 353.76, 354.56, 355.37, 356.17, 356.97, 357.77, 358.57, 359.37, 360.18, 360.98, 361.78, 362.58, 363.38, 364.18, 364.98, 365.78, 366.58, 367.38, 368.18, 368.98, 369.78, 370.58, 371.38, 372.17, 372.97, 373.77, 374.57, 375.37, 376.17, 376.96, 377.76, 378.56, 379.36, 380.15, 380.95, 381.75, 382.54, 383.34, 384.14, 384.93, 385.73, 386.52, 387.32, 388.11, 388.91, 389.7, 390.5, 391.29, 392.08, 392.88, 393.67, 394.46]",
```
Comment (reedan88) [inline calibration/NUTNRM/CGINS-NUTNRM-00727__20200103.csv:8]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00727__20230829.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...3.18, 273.99, 274.79, 275.6, 276.41, 277.21, 278.02, 278.82, 279.63, 280.44, 281.24, 282.05, 282.86, 283.67, 284.47, 285.28, 286.09, 286.89, 287.7, 288.51, 289.31, 290.12, 290.93, 291.73, 292.54, 293.35, 294.16, 294.96, 295.77, 296.58, 297.38, 298.19, 299, 299.8, 300.61, 301.42, 302.23, 303.03, 303.84, 304.65, 305.45, 306.26, 307.07, 307.87, 308.68, 309.49, 310.29, 311.1, 311.91, 312.71, 313.52, 314.33, 315.13, 315.94, 316.75, 317.55, 318.36, 319.17, 319.97, 320.78, 321.58, 322.39, 323.2, 324, 324.81, 325.61, 326.42, 327.23, 328.03, 328.84, 329.64, 330.45, 331.25, 332.06, 332.86, 333.67, 334.47, 335.28, 336.08, 336.89, 337.69, 338.5, 339.3, 340.11, 340.91, 341.71, 342.52, 343.32, 344.13, 344.93, 345.73, 346.54, 347.34, 348.14, 348.95, 349.75, 350.55, 351.35, 352.16, 352.96, 353.76, 354.56, 355.37, 356.17, 356.97, 357.77, 358.57, 359.37, 360.18, 360.98, 361.78, 362.58, 363.38, 364.18, 364.98, 365.78, 366.58, 367.38, 368.18, 368.98, 369.78, 370.58, 371.38, 372.17, 372.97, 373.77, 374.57, 375.37, 376.17, 376.96, 377.76, 378.56, 379.36, 380.15, 380.95, 381.75, 382.54, 383.34, 384.14, 384.93, 385.73, 386.52, 387.32, 388.11, 388.91, 389.7, 390.5, 391.29, 392.08, 392.88, 393.67, 394.46]",
```
Comment (reedan88) [inline calibration/NUTNRM/CGINS-NUTNRM-00727__20230829.csv:8]: Looks good.

**Review by reedan88 — APPROVED** [review]
Spot checked values. Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01086__20230830.csv (hunk 1 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01086__20230830.csv:8]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01087__20230830.csv (hunk 2 above)
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01087__20230830.csv:8]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00727__20200103.csv (hunk 3 above)
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00727__20200103.csv:8]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00727__20230829.csv (hunk 4 above)
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00727__20230829.csv:8]: looks good

**Review by cdobs — APPROVED** [review]
looks good

**Pull request merged**